tests/4.5/target_enter_exit_data/test_target_enter_exit_data_if.c

```diff
@@ -37,10 +38,13 @@ int main() {
   }
 
   // We test for offloading
-#pragma omp target map(from: isOffloading)
-  {
-    isOffloading = !omp_is_initial_device();
-  }
+  int isOffloading; 
+  OMPVV_TEST_AND_SET_OFFLOADING(isOffloading);
+ 
+  if (!isOffloading)
+  OMPVV_WARNING("It is not possible to test conditional data transfers "
```

Comment:
use OMPVV_WARNING_IF(!isOffloading). And double check if the shared env is also true here.